### PR TITLE
CMake: find prometheus(fix #1304)

### LINKED ISSUE #1304: CMake Build doesn't detect Prometheus
I've noticed that the CMake build doesn't seem to detect Prometheus libraries, although configure can.

It seems like a script to find it would be needed based on the output from the CMake github automation: https://github.com/coturn/coturn/actions/runs/6622872744/job/17989011141#step:5:106  (note that the github automation does install prometheus, so it would be expected to find it)

There's some other libraries listed that also don't have a find script. Sorry for no PR on this, CMake scripting is outside my wheelhouse.

### PR DESCRIPTION
**Pull request by KangLin**
fix #1304

### CODE REVIEW

File: .github/workflows/cmake.yaml [hunk 1]
```diff
@@ -2,7 +2,6 @@ name: CMake
 
 on:
   push:
-    branches: [ $default-branch ]
```
Comment (eakraly) [inline .github/workflows/cmake.yaml:5]: Why change it?

Comment (KangLin) [reply .github/workflows/cmake.yaml:5]: Because need run github action in my clone.

Comment (eakraly) [reply .github/workflows/cmake.yaml:5]: Let's remove it then and submit.Thanks!

**Review by eakraly — APPROVED** [review]
Assuming the default-branch build change is reverted

**Pull request merged**